src/lib/containers/entity/annotations/AdditionalPropertiesSchemaField.tsx

```diff
@@ -0,0 +1,282 @@
+import { FieldProps, utils as rjsfUtils } from '@sage-bionetworks/rjsf-core'
+import React, { useEffect, useState } from 'react'
+import { FormControl, FormGroup, FormLabel } from 'react-bootstrap'
+import { useListState } from '../../../utils/hooks/useListState'
+import { CustomArrayFieldTemplate } from './CustomArrayFieldTemplate'
+
+// Matches ####-##-##T##:##:##.###Z, e.g. 1970-01-01T12:00:000Z
+const ISO_TIMESTAMP_REGEX = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z/
```

Comment:
Is it true that all timestamp annotations are formatted in this way?

Reply:
Date type annotations are returned as timestamps (in ms).  Does this correctly identify the type of existing annotation data?

Reply:
It's successfully matched a couple of timestamps created using the existing editor